lambda_src/drivers/process_https.py

```diff
@@ -98,10 +98,23 @@ def process_row(
     req_method: str = method.upper()
 
     if json:
+        
```

Comment:
don't add vestigial changes, esp this one since it breaks formatting guidelines from PEP-8. if it helps, install a formatter that will enforce pep8 like https://black.readthedocs.io/en/stable/